.travis.yml

```diff
@@ -29,10 +28,14 @@ env:
         # to repeat them for all configurations.
         - NUMPY_VERSION=1.9
         - ASTROPY_VERSION=stable
-        - CONDA_INSTALL='conda install -c astropy-ci-extras --yes'
-        - PIP_INSTALL='pip install'
+        - CONDA_DEPENDENCIES=''
```

Comment:
I shouldn't have removed the Cython here as there are some c files in the package-template tests. Though we may want to leave a comment that most of the packages would not need to install it.